o365-auth/src/main/java/com/microsoft/office365/auth/MSALAuthenticationCallback.java

```diff
@@ -0,0 +1,11 @@
+package com.microsoft.office365.auth;
```

Comment:
We should be able to remove this file and use the implementation already provided in the MSAL lib under `com.microsoft.identity.client.AuthenticationCallback`.

Reply:
Done! Thanks